sale_exception_credit_limit/views/res_partner_views.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- make readonly for everyone -->
+    <record id="view_credit_readonly_partner_form" model="ir.ui.view">
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="account.view_partner_property_form"/>
+        <field name="arch" type="xml">
+            <group name="sale" position="inside">
+                <field name="credit_limit" readonly="1"/>
+            </group>
+        </field>
+    </record>
+
+    <!-- Make editable for user_edit_credit_limit  -->
+    <record id="view_credit_editable_partner_form" model="ir.ui.view">
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="view_credit_readonly_partner_form"/>
+        <field name="groups_id" eval="[(6, 0, [ref('credit_config')])]"/>
```

Comment:
@ivantodorovich  should be `ref('module_name.credit_config', False)`

Reply:
Hi @nikul-serpentcs that's actually not necessary. That group is defined in this same module.

Also if you use ref(.., False) you would be silently ignoring the exception if the xmlid is not found. Why would you want to do that?

Reply:
> Also if you use ref(.., False) you would be silently ignoring the exception if the xmlid is not 
> found. Why would you want to do that?


If `xmlid is not found` then silently ignoring the exception That Good, No need to False/ `ref('module_name.credit_config', False)`.

> that's actually not necessary. That group is defined in this same module.

Yes, I know but if you put `module_name` Is Good For us.

 Otherwise, Code Look Good.